### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by Israel-Laguan**
# Resume

I've followed the specifications as described in the project requirements, added the event listeners, passed props and connect to buttons, and *came to the conclusion that I met all of the requirements*.

# Proposal

I want to have a review of the code to know if I need to make any changes. If no changes needed I want the project to be ACCEPTED so I can move to the next project with a happy smile :)

# Specifications

- Followed [these requirements](https://github.com/microverseinc/project-react-calculator/blob/master/milestones/MILESTONE_5.md) for Milestone 5
- Added state to `src/components/App.js`
- Added Event Handler function on `App.js`
- Passed props to Display, Button Panel, and each Button
- Checked all functionalities are working
- Checked `ESlint` & `Jest`
- Updated on Heroku

# [Live Version Here](https://math-magicians-calculator.herokuapp.com/)
image

# Tests

### `ESLint`
image

### `Jest`
image

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -35,4 +38,12 @@ const ButtonPanel = () => (
   </div>
 );
 
+ButtonPanel.propTypes = {
+  onClick: PropTypes.func,
+};
+
+ButtonPanel.defaultProps = {
+  onClick: e => e.target,
```
Comment (Oghenebrume50) [inline src/components/ButtonPanel.js]: There should be an handleclick function see this
calling e.target on ButtonPanel is not ideal, buttonPanel is a react component not a DOM element.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -26,6 +28,7 @@ const ButtonPanel = () => (
                 key={Math.random()}
                 color={(id !== 4 || i !== 2) && i !== 3}
                 wide={name === '0'}
+                onClick={onClick}
```
Comment (Oghenebrume50) [inline src/components/ButtonPanel.js]: [Optional] change the name of the prop onClick to better suit it's purpose may be a better name would be handleClick as suggested above from the screenshot

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED 

Hi @Israel-Laguan good job working on this, I really like what you have accomplished and everything about this calculator works fine as of now :+1: . But there is a slight change to be made :slightly_smiling_face:  

Please see it below, fix and request a re-review :slightly_smiling_face:  
I am available on slack, for any questions 

**RAPHAEL NORIODE** :sunglasses:

File: src/components/ButtonPanel.js (hunk 1 above)
Comment (Israel-Laguan) [reply src/components/ButtonPanel.js]: Thanks for your review. I'll change those details ASAP

**Review by bellom — APPROVED** [review]
## Hi @Israel-Laguan  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (Israel-Laguan) [reply src/components/ButtonPanel.js]: Done!

File: src/components/ButtonPanel.js (hunk 1 above)
Comment (Israel-Laguan) [reply src/components/ButtonPanel.js]: Done!

**Comment by uptowndat777777** [comment]
Hello, Compliment of the day, how are you today ? i am Happy to get together with you,My name is Serach , I'm a military medical nurse from israel. Please let not my profession, race or nationality inch your mind, I have gone across your profile and became interested in you, we can be good friends, contact me through my private email address/  //abelmans@aol.com   ////

**Pull request merged**